pkg/internal/apis/certmanager/types_certificate.go

```diff
@@ -59,13 +59,12 @@ const (
 
 // CertificateSpec defines the desired state of Certificate
 type CertificateSpec struct {
+	// A valid Certificate requires at least one of a CommonName, DNSName, or
+	// URISAN to be valid.
```

Comment:
This comment should be put *outside* the struct so it is picked up and inserted into our reference docs 😄